src/mgmt/srv_system.c

```diff
@@ -343,37 +343,31 @@ ds_mgmt_svc_start(void)
 	return rc;
 }
 
+static void
+stopper(void *arg)
+{
+	int rc;
+
+	rc = ds_rsvc_stop_nodb(DS_RSVC_CLASS_MGMT, &mgmt_svc_id);
+	if (rc != 0)
+		D_DEBUG(DB_MGMT, "ignoring "DF_RC"\n", DP_RC(rc));
```

Comment:
I wonder if this should be D_ERROR().

Reply:
In an engine who hasn't started mgmt_svc, rc will be -DER_ALREADY when we reach here. I assume NLT would complain if we logged this as an error, no?

Reply:
As long as NLT is consistent then there's no problem there.

If there are legitimate reasons this code could be reached then it's not an error so D_DEBUG() is correct.